### ISSUE TITLE
Library files not found (group_shared.hpp, features.h) 

### ISSUE THREAD

**Issue by Cnordbo** [realm/realm-js#2641]
## Goals
Run the application in iOS emulator after adding a new package via yarn/npm

## Expected Results

Applications starts successfully

## Actual Results
```
❌/MyApp/node_modules/realm/src/object-store/src/thread_safe_reference.hpp:22:10: 'realm/group_shared.hpp' file not found

#include <realm/group_shared.hpp>
         ^~~~~~~~~~~~~~~~~~~~~~~~


▸ Compiling sync_permission.cpp
▸ Compiling sync_user.cpp
▸ Compiling RealmReact.mm
▸ Compiling sync_file.cpp
▸ Compiling system_configuration.cpp

❌/MyApp/node_modules/realm/src/object-store/src/sync/impl/network_reachability.hpp:22:10: 'realm/util/features.h' file not found

#include <realm/util/features.h>
```

## Steps to Reproduce
Successfully implement Realm to your React-native application. 

Add a second package that requires pods to be installed, run pod install. 

Try to run the application again with `yarn/npx react-native run-ios`

## Version of Realm and Tooling

- Realm JS SDK Version: 3.5.0
- React Native: 0.61.5
- Client OS & Version: MacOS Catalina (10.15.2)
- Which debugger for React Native: None

**Comment by Cnordbo** [comment]
##Workaround
Delete `ios/pods` and run `pod install` again.

**Comment by kneth** [comment]
@Cnordbo Another workaround is outlines in https://github.com/realm/realm-js/issues/2617#issuecomment-559446483. 

I close the issue for now, please follow #2671 to see when we have a proper solution.

**Comment by Cnordbo** [comment]
Thanks, error messages just seem to differ - which explains why it didnt show up in search.

**Comment by kraenhansen** [comment]
@Cnordbo a fix for the #2617 issue was just released as Realm JS v2.6.0, please upgrade and verify that it fix your issue too. If not - please reopen this or open a new issue. Thanks!

**Comment by Cnordbo** [comment]
Issue seems to be resolved! Thanks a lot for the quick update!
